website/source/guides/advanced-scheduling/affinity.html.md

```diff
@@ -0,0 +1,217 @@
+
+### Step 3: Register the Job `redis.nomad`
+
+Run the Nomad job with the following command:
+
+```shell
+$ nomad run redis.nomad 
+==> Monitoring evaluation "11388ef2"
+    Evaluation triggered by job "redis"
+    Allocation "0dfcf0ba" created: node "6b6e9518", group "cache1"
+    Allocation "89a9aae9" created: node "3592943e", group "cache1"
+    Allocation "9a00f742" created: node "6b6e9518", group "cache1"
+    Allocation "fc0f21bc" created: node "3dea0188", group "cache1"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "11388ef2" finished with status "complete"
+```
+
+Note that two of the allocations in this example have been placed on node `6b6e9518`. This is the node we configured to be in datacenter `dc2`. The Nomad scheduler selected this node because of the affinity we specified. All of the allocations have not been placed on this node because the Nomad scheduler considers other factors in the scoring such as bin packing. This helps avoid placing too many instances of the same job on a node and prevents reduced capacity during a node level failure. We will take a detailed look at the scoring in the next few steps.
+
+### Step 4: Check the Status of the `redis` Job
+
+At this point, we are going to check the status of our job and verify where our
+allocations have been placed. Run the following command:
+
+```shell
+$ nomad status redis
+```
+
+You should see 4 instances of your job running in the `Summary` section of the
+output as show below:
+
+```shell
+...
+Summary
+Task Group  Queued  Starting  Running  Failed  Complete  Lost
+cache1      0       0         4        0       0         0
+
+Allocations
+ID        Node ID   Task Group  Version  Desired  Status   Created    Modified
+0dfcf0ba  6b6e9518  cache1      0        run      running  1h44m ago  1h44m ago
+89a9aae9  3592943e  cache1      0        run      running  1h44m ago  1h44m ago
+9a00f742  6b6e9518  cache1      0        run      running  1h44m ago  1h44m ago
+fc0f21bc  3dea0188  cache1      0        run      running  1h44m ago  1h44m ago
+```
+
+As stated earlier, you can cross-check this output with the results of the
+`nomad node status` command to verify that the majority of your workload has
+been placed on the node in `dc2` (in our case, that node is `6b6e9518`).
+
+### Step 5: Obtain Detailed Scoring Information on Job Placement
+
+As stated earlier, the Nomad scheduler will not necessarily place all of your workload on nodes you have specified in the `affinity` stanza even if the resources are available. This is because affinity scoring is combined with other metrics as well before making a scheduling decision. In this step, we will take a look at some of those other factors.
```

Comment:
replace "not necessarily" with either "is not guarenteed to place" or "will not always place", here and in other places